_config.yml

```diff
@@ -134,7 +134,7 @@ subscribeInfo: "Registration will be open around end of August. Number of ticket
 
 # Tickets Block
 ticketsTitle: "Registration"
-ticketsInfo: 'Tickets grant access to all sessions, lunch on both day and one conference dinner. Accommodation is NOT included in the ticket price. If you would like to apply for a waiver, please send an email to <a href="mailto:secretary@nordic-rse.org">secretary@nordic-rse.org</a>, decision will be made on a per-case basis.'
+ticketsInfo: 'Tickets grant access to all sessions, lunch on both day and one conference dinner. Accommodation is NOT included in the ticket price. If you would like to apply for a waiver, please send an email to <a href="mailto:secretary@nordic-rse.org">secretary@nordic-rse.org</a>, decision will be made on a per-case basis. <b>Make sure to register by April 29th!</b>'
```

Comment:
```suggestion
ticketsInfo: 'Tickets grant access to all sessions, lunch on both day and one conference dinner. Accommodation is NOT included in the ticket price. If you would like to apply for a waiver, please send an email to <a href="mailto:secretary@nordic-rse.org">secretary@nordic-rse.org</a>, decision will be made on a per-case basis. <b>Registration will be open until April 29th or until we reach capacity of the venue!</b>'
```

Reply:
should we mention how much full capacity is?

Reply:
Do we know? :D

Reply:
I guess partly it is also about "how many we can handle" apart from the conference venue :thinking:

Reply:
But yea, agree there should probably be some number, maybe we could add "(approximately X)" with X being the max people allowance for our main room, @mtomasini ?

Reply:
maybe let's merge this and clarify later if needed?